lib/db_utils.go

```diff
@@ -476,8 +476,37 @@ type DBPrefixes struct {
 	// 	<prefix, expirationBlockHeight, PKID, partialID> -> <>
 	PrefixNoncePKIDIndex []byte `prefix_id:"[77]" is_state:"true"`
 
-	// NEXT_TAG: 78
+	// PrefixValidatorByPKID: Retrieve a validator by PKID.
+	// Prefix, ValidatorPKID -> ValidatorEntry
+	PrefixValidatorByPKID []byte `prefix_id:"[78]" is_state:"true"`
 
+	// PrefixValidatorByStake: Retrieve the top N validators by stake.
+	// Prefix, TotalStakeAmountNanos, MaxUint64 - RegisteredAtBlockHeight, ValidatorPKID -> ValidatorPKID
+	// FIXME: @DH, should we duplicate the ValidatorPKID in the key and the value?
+	// Alternatively, we could just store and parse the ValidatorPKID from the key
+	// and store a struct{} as the value. That saves on space, but makes retrieving
+	// the ValidatorPKID from the key bytes more complex than just reading the value
+	// bytes directly since the key includes other preceding fields. Interesting
+	// trade-off. Curious your opinion.
+	PrefixValidatorByStake []byte `prefix_id:"[79]" is_state:"true"`
+
+	// PrefixGlobalStakeAmountNanos: Retrieve the cumulative stake across all validators.
+	// Prefix -> *uint256.Int
+	PrefixGlobalStakeAmountNanos []byte `prefix_id:"[80]" is_state:"true"`
+
+	// PrefixStakeByValidatorByStaker: Retrieve a StakeEntry.
+	// Prefix, ValidatorPKID, StakerPKID -> StakeEntry
+	PrefixStakeByValidatorByStaker []byte `prefix_id:"[81]" is_state:"true"`
```

Comment:
Nitpick: We normally use "By" to denote -> and "And" to denote (A, B, C). So in this case, since you have (ValidatorPKID, StakerPKID) -> StakeEntry, I would call it `PrefixStakeByValidatorAndStaker`.

I would make the same change to the other things.

Reply:
I fixed it for you btw. Just look at how I did it for future reference and resolve.